lib/src/blurhash.dart

```diff
@@ -105,8 +109,7 @@ class BlurHash {
         'BlurHash components must be between 1 and 9.',
       );
     }
-
-    final data = image.getBytes(format: Format.rgba);
+    final data = image.getBytes();
```

Comment:
What about different channel orderings? Can this handle e.g. `bgra` images? 
If there is no good way to support that, I would at least throw an exception to verify the input image format.